### PR TITLE
Notify players in deadchat when the game ends

### PR DESCRIPTION
**Pull request by cyberfawkes**
Added a PM to inform Deadchat Players that the game has ended.

### CODE REVIEW

File: src/wolfgame.py [hunk 1]
```diff
@@ -2440,7 +2440,9 @@ def stop_game(cli, winner = "", abort = False, additional_winners = None):
 
     if not abort:
         cli.msg(chan, gameend_msg)
-
+        # Message Players in Deadchat letting them know that the game has ended. cyberfawkes
+        mass_privmsg(cli, var.DEADCHAT_PLAYERS, messages["endgame_deadchat"].format(chan))
```
Comment (skizzerz) [inline src/wolfgame.py]: This should be moved to after the end-of-game stuff has been run, but before reset(). Displaying the list of winners and recording the stats is more important than letting the people in deadchat know the game is over. Also, remove your name from the comment. Nobody "owns" the code, and git already supplies attribution information.

Comment (nyuszika7h) [inline src/wolfgame.py]: Nitpick: please Get rid Of unnecessary Capitalization. :P

Comment (cyberfawkes) [inline src/wolfgame.py]: I'll fix it right now, I forgot to remove my name. I left it in there for debugging so I didn't have to remember the specific line that line of code was on, I could just search "cyberfawkes".

**Comment by nyuszika7h** [comment]
Merged as 71ff8c3.

**Pull request closed without merging**